test/integration/utility.h

```diff
@@ -192,13 +192,11 @@ class IntegrationUtil {
 
   /**
    * Create transport socket factory for Quic upstream transport socket.
-   * @param context supplies the port to connect to on localhost.
-   * @param san_to_match configs |context| to match Subject Alternative Name during certificate
```

Comment:
this comment is still valid?

Reply:
yep.  I took out args which no longer existed though didn't add comments for things others had added as it's only a test util.